api.js

```diff
@@ -99,10 +99,25 @@ module.exports = function(_roommanager, storage) {
 			});
 			return;
 		}
-		let success = roommanager.addToQueue(req.params.name, req.body.url);
-		res.json({
-			success
-		});
+		if (req.body.url) {
```

Comment:
Is there any reason why you can't create a response which just leaves null values (empty string in JSON) for parameters that are not provided? I don't think it's a good idea to mix data types when it comes to API endpoints.